src/lib/Server/Controller/Location.php

```diff
@@ -279,6 +280,39 @@ public function moveSubtree($locationPath, Request $request)
         }
     }
 
+    /**
+     * @throws \Ibexa\Contracts\Core\Repository\Exceptions\NotFoundException
+     * @throws \Ibexa\Contracts\Core\Repository\Exceptions\InvalidArgumentException
+     * @throws \Ibexa\Contracts\Core\Repository\Exceptions\UnauthorizedException
+     */
+    public function moveLocation(Request $request, string $locationPath): Value
```

Comment:
When there is only one possible type returned, you can narrow the return type down (even and especially if you implement an interface or extend a class).

This is available since PHP 7.4 (ish, don't quote me on that).

```suggestion
    public function moveLocation(Request $request, string $locationPath): Values\ResourceCreated
```